migrations/kruize_experiments_ddl.sql

```diff
@@ -1,4 +1,6 @@
-create table IF NOT EXISTS kruize_experiments (experiment_id varchar(255) not null, cluster_name varchar(255), datasource jsonb, experiment_name varchar(255), extended_data jsonb, meta_data jsonb, mode varchar(255), performance_profile varchar(255), status varchar(255), target_cluster varchar(255), version varchar(255), primary key (experiment_id));
+create table IF NOT EXISTS kruize_datasources (version varchar(255), name varchar(255), provider varchar(255), serviceName varchar(255), namespace varchar(255), url varchar(255), primary key (name));
+create table IF NOT EXISTS kruize_metadata (id serial, version varchar(255), cluster_group_name varchar(255), cluster_name varchar(255), namespace varchar(255), workload_type varchar(255), workload_name varchar(255), container_name varchar(255), container_image_name varchar(255), primary key (id));
```

Comment:
add  ddl's related to Kruize local into different file 
refer https://github.com/kruize/autotune/pull/1154

Reply:
If both ddls shares same table name then have alter ddls

for example Kruize_experiments available in both  then have alter commands

Reply:
Updated.